www/_template/tutorials/svelte.md

```diff
@@ -0,0 +1,327 @@
+
+```diff
+<!-- src/App.svelte -->
+
+  <header class="App-header">
++   <img src="/logo.svg" class="App-logo" alt="logo" />
+    <a
+      class="App-link"
+      href="https://svelte.dev"
+      target="_blank"
+      rel="noopener noreferrer">
+      Learn Svelte
+    </a>
+```
+
+<div class="frame"><img src="/img/guides/svelte/svelte-logo-snowpack.jpg" alt="Side by side of code and site. The site now has a very large Svelte logo. The code shows the src/App.svelte file "  class="screenshot"/></div>
+
+With Svelte, CSS can go directly in your `.svelte` component. Add this code to the top of `App.svelte` between the `<style>` tags:
+
+```html
+<!-- src/App.svelte -->
+
+<style>
+  .App-header {
+    background-color: #f9f6f6;
+    color: #333;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+  }
+  .App-logo {
+    height: 36vmin;
+    pointer-events: none;
+    margin-bottom: 3rem;
+    animation: App-logo-pulse infinite 1.6s ease-in-out alternate;
+  }
+  @keyframes App-logo-pulse {
+    from {
+      transform: scale(1);
+    }
+    to {
+      transform: scale(1.06);
+    }
+  }
+</style>
+```
+
+<div class="frame"><img src="/img/guides/svelte/svelte-logo-style-snowpack.gif" alt="code and site side by side, when the css is added to the Svelte component, the background becomes a beige, the logo shrinks down, and the logo has a pulsing animation" class="screenshot"/></div>
+
+## Adding a counter to your Svelte component
+
+Snowpack is one of the only Svelte dev environments to support Fast Refresh by default. As you make changes to `.svelte` files, Snowpack updates the browser for you without losing your place or resetting component state. That's Fast Refresh in action. To see this for yourself, go ahead and add a simple timer to your App.svelte component.
+
+Svelte components include component specific scripts in a `<script>` tag. Add the counter here in `App.svelte` between the `<script>` tags:
+
+```html
+<!-- src/App.svelte -->
+
+<script>
+  import { onMount } from 'svelte';
+  let count = 0;
+  onMount(() => {
+    const interval = setInterval(() => count++, 1000);
+    return () => {
+      clearInterval(interval);
+    };
+  });
+</script>
+```
+
+Then lower down in your component's body, add this code that displays the results of the timer.
+
+```diff
+<!-- src/App.svelte -->
+
+<div class="App">
+    <header class="App-header">
+      <img src="/logo.svg" class="App-logo" alt="logo" />
++     <p>Page has been open for <code>{count}</code> seconds.</p>
+      <a class="App-link" href="https://svelte.dev" target="_blank" rel="noopener noreferrer">
+        Learn Svelte
+      </a>
+    </header>
+</div>
+```
+
+Trying making a change to the code. You'll see the timer does not reset.
```

Comment:
We've seen some users get confused when they change `count = ...` and HMR doesn't save state (by design). To protect from this, lets give a specific suggestion here, like:

> Change some code on the page (like the "Learn Svelte" button)